package.json

```diff
@@ -2,6 +2,9 @@
   "name": "ionic-docs",
   "version": "0.0.0",
   "private": true,
+  "engines": {
+    "node": ">=18.0.0"
+  },
```

Comment:
This can be removed once the project settings on Vercel are updated to Node v18.